style.css

```diff
@@ -0,0 +1,222 @@
+html {
+    box-sizing: border-box;
+}
+
+*, *:before, *:after {
+    box-sizing: inherit;
+}
+
+body {
+    background-color: #ECEEF1;
+    font-family: AvenirNext,Avenir,"Helvetica Neue",Helvetica,Arial,sans-serif;
+}
+
+#main-wrapper {
+    display: flex;
+    flex-direction: column;
+}
+
+header {
+    width: 100vw;
+    height: 10vh;
```

Comment:
Kudos for using `vh` units but for this particular case it resolves into a lot of wasted space on the header. You can omit the assignment of height

Reply:
Removed height: 10vh